tests/rhosak/kas_with_instance.spec.ts

```diff
@@ -177,13 +177,57 @@ test('test instance dashboard on instance name click', async ({ page }) => {
   await expect(page.getByTestId('pageKafka-tabDashboard')).toHaveCount(1);
 });
 
+// test_4kas.py test_kafka_dashboard_default
+test('test kafka dashboard default', async ({ page }) => {
+  await waitForKafkaReady(page, testInstanceName);
+  await page.locator('a', { hasText: `${testInstanceName}` }).click();
+  await expect(page.locator('h1', { hasText: `${testInstanceName}` })).toHaveCount(1);
+  await expect(page.locator('button').locator('span', { hasText: 'Dashboard' })).toHaveCount(1);
+  await expect(page.locator('h3', { hasText: 'Topics' })).toHaveCount(1);
+  await expect(page.locator('h3', { hasText: 'Topic partitions' })).toHaveCount(1);
+  await expect(page.locator('h3', { hasText: 'Consumer groups' })).toHaveCount(1);
+  await expect(page.locator('h3', { hasText: '0' })).toHaveCount(3);
+});
+
 // test_4kafka.py test_kafka_topic_create
 test('create and delete a Kafka Topic', async ({ page }) => {
   await navigateToKafkaTopicsList(page, testInstanceName);
   await createKafkaTopic(page, testTopicName);
   await deleteKafkaTopic(page, testTopicName);
 });
 
+// test_4kafka.py test_kafka_topics_opened
+test('test kafka topics opened', async ({ page }) => {
+  await waitForKafkaReady(page, testInstanceName);
+  await page.locator('a', { hasText: `${testInstanceName}` }).click();
+  await page.locator('button[aria-label="Topics"]').click();
+  await expect(page.locator('h2', { hasText: 'No topics' })).toBeVisible();
+  await expect(page.locator('button', { hasText: 'Create topic' })).toBeVisible();
+});
+
+// test_4kafka.py test_kafka_topic_check_does_not_exist
+test('test kafka topic check does not exist', async ({ page }) => {
+  await waitForKafkaReady(page, testInstanceName);
+  await page.locator('a', { hasText: `${testInstanceName}` }).click();
+  await expect(page.locator('h1', { hasText: `${testInstanceName}` })).toHaveCount(1);
+  await page.locator('button[aria-label="Topics"]').click();
+  // expecting not to find topic row
+  await expect(page.getByText(testTopicName)).toBeHidden();
```

Comment:
Just a nitpick, have you tried `.not.toBeVisible();` here? Just to know how it would behave... 
https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-hidden
> Ensures that [Locator](https://playwright.dev/docs/api/class-locator) either does not resolve to any DOM node, **_or resolves to a [non-visible](https://playwright.dev/docs/actionability#visible) one._**

Reply:
That approach did not pass linting and I had to commit the change

Reply:
Can I resolve this conversation @mmusil ?

Reply:
How it could not have passed linting? Are you sure that was the problem and not something else? I just tried to copy&paste that test, change it to what I mentioned and linting passed fine 🤔

Reply:
not on this PR Jenkins pipeline

Reply:
Can we resolve this conversation @Frawless @mmusil ?

Reply:
I am pretty sure the issue wasn't caused by it. Could you try to use a different locator? I mean get rid of getByText? After that you could use `count()` to check expected result

Reply:
changed @Frawless Can we resolve this conversation?